sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClient.java

```diff
@@ -257,8 +259,10 @@ public TextAnalyticsServiceVersion getServiceVersion() {
         }
         final String finalCountryHint = countryHint;
         return detectLanguageBatch(
-            mapByIndex(documents, (index, value) ->
-                new DetectLanguageInput(index, value, finalCountryHint)), options);
+            mapByIndex(documents, (index, value) -> {
```

Comment:
@iscai-msft  Ahh. Sorry. I forget to remove this part from my PR. I have think to the null checking when creating TextLanguageInput/DetectLanguageInput.